addons/populate/functions/fnc_createPatrols.sqf

```diff
@@ -0,0 +1,57 @@
+#include "script_component.hpp"
+/*
+    Author:
+        Bridg
+
+    Description:
+        Spawns a unit at the passed position
+
+    Parameters:
+        0: _numberOfUnitsToSpawn <NUMBER>
+        1: _module <OBJECT>
+        2: _group <GROUP>
+
+    Return value:
+        Nothing
+*/
+
```

Comment:
Could simplify this quite a bit by making use of `common,spawnGroupInfantry`.
Take the nubmer of units remaining and divide by the patrol size. Do that many iterations plus one more for the remainder.
The spawning can be staggered by using a waitAndExecute with the time based on the index of an array multiplied by seconds